input/data/2014/2014ApJ...782...13A/tev-000013-sed-1.ecsv

```diff
@@ -0,0 +1,29 @@
+# %ECSV 0.9
+# ---
+# datatype:
+# - {name: e_ref, datatype: float32}
+# - {name: e_min, datatype: float32}
+# - {name: e_max, datatype: float32}
+# - {name: dnde, unit: TeV-1 m-2 s-1, datatype: float32}
+# - {name: dnde_err, unit: TeV-1 m-2 s-1, datatype: float32}
+# - {name: dnde_ul, unit: TeV-1 m-2 s-1, datatype: float32}
+# - {name: significance, datatype: float32}
+# meta: !!omap
+# - data_type: sed
+# - source_id: 13
+# - file_id: 1
+# - reference_id: 2014ApJ...782...13A
+# - telescope: veritas
```

Comment:
```
ERROR:gammacat.sed:SED file /Users/deil/code/gamma-cat/input/data/2014/2014ApJ...782...13A/tev-000013-sed-1.ecsv contains an upper limit column, but not "UL_CONF" in meta.
```
Do you know the confidence level of the upper limits given here?
If yes, please add as `UL_CONF` header key.
If no, please add a comment in the header saying `UL_CONF` is unknown for this paper.